packages/grafana-data/src/types/templateVars.ts

```diff
@@ -141,15 +141,15 @@ export interface DashboardProps {
   toString: () => string;
 }
 
```

Comment:
Please revert these changes.

Reply:
Sorry， auto format of my VSCODE，I will revert all auto format changed